tests/queries/0_stateless/02114_bool_type.sql

```diff
@@ -0,0 +1,38 @@
+DROP TABLE IF EXISTS bool_test;
+
+CREATE TABLE bool_test (value Bool) ENGINE = Memory;
+
+-- value column shoud have type 'Bool'
+SHOW CREATE TABLE bool_test;
+
+INSERT INTO bool_test (value) VALUES ('false'), ('true'), (0), (1);
+INSERT INTO bool_test (value) FORMAT JSONEachRow {"value":false}{"value":true}{"value":0}{"value":1}
+
+SELECT value FROM bool_test;
+SELECT value FROM bool_test FORMAT JSONEachRow;
+SELECT toUInt64(value) FROM bool_test;
+SELECT value FROM bool_test where value > 0;
+
+set bool_true_representation='True';
+set bool_false_representation='False';
+INSERT INTO bool_test (value) FORMAT CSV True
+INSERT INTO bool_test (value) FORMAT TSV False
+SELECT value FROM bool_test order by value FORMAT CSV;
+SELECT value FROM bool_test order by value FORMAT TSV;
+
+set bool_true_representation='Yes';
+set bool_false_representation='No';
+INSERT INTO bool_test (value) FORMAT CSV Yes
```

Comment:
Need to add a test with more than one column in a table.

Reply:
ok,i will add more test

Reply:
i have some update